aip/general/0193.md

```diff
@@ -44,8 +44,13 @@ strings **may** change over time; however, if an error message does not have a
 machine-readable identifier _in addition to_ the `code` field, changing the
 error message **must** be considered a backwards-incompatible change.
 
-**Note:** The [`ErrorInfo`][ErrorInfo] message is the recommended way to send a
-machine-readable identifier.
+### ErrorInfo
+
+The [`ErrorInfo`][ErrorInfo] message is the required way to send a
+machine-readable identifier. All error responses **must** include an
+`ErrorInfo` payload in the `details` field. Variable information
+**should** be included in the `metadata` field on `ErrorInfo` and
+**must** be included if it appears within an error message.
 
 #### Uniqueness
```

Comment:
Update this section to say there **must** be exactly one ErrorInfo, and not more than one of other details.  

At the moment it says **must not** be more than one ErrorInfo.